packages/fast-usdc-deploy/test/fast-usdc.test.ts

```diff
@@ -367,10 +367,16 @@ test.serial('writes pool metrics to vstorage', async t => {
   await documentStorageSchema(t, storage, doc);
 });
 
-test.serial('deploy HEAD; update Settler reference', async t => {
+test.serial('deploy RC2; update Settler reference', async t => {
+  const { evalReleasedProposal } = t.context;
+  await evalReleasedProposal('fast-usdc-rc2', 'eval-fast-usdc-settler-ref');
+  t.pass();
```

Comment:
t.pass() is rarely optimal.
```suggestion
  t.deepEqual(actual, { incarnationNumber: 3 });
```

Reply:
Neither `evalReleasedProposal` nor `evalProposal` seem to return anything - just undefined.

I amended to wrap in `t.notThrowsAsync()`, although I'm not sure that correct either. It doesn't seem like either will throw if the proposal is rejected